internal/backends/winit/winitwindowadapter.rs

```diff
@@ -392,6 +392,27 @@ impl WinitWindowAdapter {
     ) -> Result<WindowAttributes, PlatformError> {
         let mut attrs = WindowAttributes::default().with_transparent(true).with_visible(false);
 
+        #[cfg(all(unix, not(target_os = "macos")))]
+        {
+            if let Some(xdg_app_id) = i_slint_core::with_global_context(
```

Comment:
Would be nice to use `WindowInner::ctx`  instead of the global context.  (For Qt too)

Reply:
That was my first attempt, but the issue is that doing `app.window().set_xdg_app_id(...)` happens too late. After the call to `window()` we've already created the window adapter and thus the window, and winit doesn't support setting the app id afterwards - it's propagated in the `WindowAttributes`.

Reply:
I meant to get it.
`WindowInner::from_pub(self.window).ctx.xdg_app_id()` or something like that. You'll have to work around privacy rule. But the idea is to use the SlintContext associated with the window rather than the global one. (Even if for now it's the same, but ideally we could be able to run Slint without a global context in the future)

Or are you saying that the window is not yet associated with a context at this point?

Reply:
Ohh, I missed that. Thanks! I'll try that.